### PR TITLE
Add 'relative: bool' to tanArcTo, remove 'to' mode from tanArc

### PR DESCRIPTION
**Pull request by adamchalmers**
Fixes https://github.com/KittyCAD/modeling-app/issues/3319

### CODE REVIEW

**Comment by jtran** [comment]
Pretty sure these need to get updated.

https://github.com/KittyCAD/modeling-app/blob/1162ff3b03a5f0179c47d57d36a77c113b78b708/src/lang/std/sketch.ts#L852-L853

The argument defaults to `2`, but this PR adds a new parameter. So I think it needs to change to 3. It's which parameter is possibly the TagDeclarator.

File: src/wasm-lib/kcl/src/std/args.rs [hunk 1]
```diff
@@ -558,7 +561,7 @@ impl<'a> FromKclValue<'a> for &'a str {
 
 impl<'a> FromKclValue<'a> for TagDeclarator {
     fn from_mem_item(arg: &'a KclValue) -> Option<Self> {
-        arg.get_tag_declarator().ok()
+        dbg!(arg.get_tag_declarator()).ok()
```
Comment (jtran) [inline src/wasm-lib/kcl/src/std/args.rs:564]:
```suggestion
        arg.get_tag_declarator().ok()
```

**Comment by jtran** [comment]
Sorry, I meant when we call it on these lines, the argument should be `3`. The default of the function shouldn't change.

https://github.com/KittyCAD/modeling-app/blob/1162ff3b03a5f0179c47d57d36a77c113b78b708/src/lang/std/sketch.ts#L852-L853

File: src/lang/std/sketch.ts [hunk 2]
```diff
@@ -309,7 +309,7 @@ function singleRawValueHelper(
   ]
 }
 
-function getTag(index = 2): SketchLineHelper['getTag'] {
+function getTag(index = 3): SketchLineHelper['getTag'] {
```
Comment (jtran) [inline src/lang/std/sketch.ts:312]: Sorry if I wasn't clear. I think we want to revert this change because it applies to all AST functions, not just tanArcTo.

```suggestion
function getTag(index = 2): SketchLineHelper['getTag'] {
```

File: src/lang/std/sketch.ts [hunk 3]
```diff
@@ -1860,7 +1861,7 @@ function isAngleLiteral(lineArugement: Expr): boolean {
 
 type addTagFn = (a: AddTagInfo) => { modifiedAst: Program; tag: string } | Error
 
-function addTag(tagIndex = 2): addTagFn {
+function addTag(tagIndex = 3): addTagFn {
```
Comment (jtran) [inline src/lang/std/sketch.ts:1864]: Revert here also.
```suggestion
function addTag(tagIndex = 2): addTagFn {
```

File: src/lang/std/sketch.ts [hunk 4]
```diff
@@ -849,8 +850,8 @@ export const tangentialArcTo: SketchLineHelper = {
       pathToNode,
```
Comment (jtran) [inline src/lang/std/sketch.ts:850]: Above and below here are AST mod stuff specific to `tangentialArcTo`, so I suspect we need to change more stuff in this block.

**Comment by adamchalmers** [comment]
Replaced by https://github.com/KittyCAD/modeling-app/pull/3729/files
